### PR TITLE
Fix formatting in I-lettered articles

### PR DESCRIPTION
**Pull request by TPGPL**

### CODE REVIEW

File: wiki/Installation/en.md [hunk 1]
```diff
@@ -6,14 +6,14 @@ needs_cleanup: true
...
+This page will tell you how to get osu! working on your Windows device. The [osu!academy](/wiki/osu!academy) has a [video tutorial](https://youtube.com/watch?v=0V5GwzmMhpU) that explains how to install osu! on Windows.
 
 ## Minimum system requirements
 
-- .NET framework 4+ (the installer should prompt you to download it, otherwise, you can [find it here](https://www.microsoft.com/en-us/download/details.aspx?id=48130))
+- .NET framework 4+ (the installer should prompt you to download it, otherwise, you can [find it here](https://microsoft.com/en-us/download/details.aspx?id=48130))
 - A decent graphics card.
 
-## Installing osu!
+## Installing osu!ss
```
Comment (MegaApplePi) [inline wiki/Installation/en.md]:
```suggestion
## Installing osu!
```

**Comment by peppy** [comment]
What is this? This should be automated and only done with CI in place. Applying this manually is a waste of review time.

File: wiki/Installation/pl.md [hunk 2]
```diff
@@ -6,28 +6,28 @@ outdated: true
 
 *Zobacz także: [Instalacja/macOS](/wiki/Installation/macOS)*
 
-Ten artykuł przybliży ci proces instalacji osu! na urządzenia z systemem Windows. [Akademia osu!](/wiki/osu!academy) podjęła ten temat w swoim [poradniku wideo](https://www.youtube.com/watch?list=PLmWVQsxi34bMYwAawZtzuptfMmszUa_tl&v=cz522ZAs5aQ).
+Ten artykuł przybliży ci proces instalacji osu! na urządzenia z systemem Windows. [Akademia osu!](/wiki/osu!academy) podjęła ten temat w swoim [poradniku wideo](https://youtube.com/watch?v=0V5GwzmMhpU).
 
 ## Minimalne wymagania systemowe
 
-- .NET Framework 4+ (możesz znaleźć go [tutaj](https://www.microsoft.com/en-us/download/details.aspx?id=48130); instalator powininen automatycznie powiadomić cię o potrzebie zainstalowania tej platformy)
```
Comment (peppy) [inline wiki/Installation/pl.md]: what is this? why are you removing www from urls?

Comment (TPGPL) [reply wiki/Installation/pl.md]: because it's not needed in any way, so why should it be there?

Comment (peppy) [reply wiki/Installation/pl.md]: oh my god.

**Comment by peppy** [comment]
Please reopen applying changes to every page with an attached automation script (at very least) and proposed plan for CI. Any manual (non-formatting) changes should be PR'd separately from ones which can be automated.

As an aside, you should NOT remove `www.` from URLs because that's not how the internet works. Please read up on this if you think you know better.

**Comment by TPGPL** [comment]
Since we have a direction to go now,  could we have this merged at least? It's still better than what we have currently and has many changes that can't be automated. It would be a waste to let it stay closed.

**Comment by peppy** [comment]
Please open in a new PR with the following changes:

- No formatting fixes (leave that for an automated pass)
- Don't remove `www` for URLs
- Make your PRs in more digestable chunks. None of this "fix i articles". Either do it one per article, folder, language or any combination. The title should be something like "Fix outdated links in Beatmaps articles".

File: wiki/Installation/pl.md (hunk 2 above)
Comment (cl8n) [reply wiki/Installation/pl.md]: in this case the canonical url does include www so you should keep it, you can tell bc without www redirects to www. sometimes the page includes a `<link rel="canonical" href="...">` to let you know the preferred url

but likewise you *shouldn't* include www if the service says it's not canonical

**Pull request closed without merging**